Jenkinsfile

```diff
@@ -19,7 +19,7 @@ node('heavy && linux && docker') {
   }
 
   withEnv(['npm_config_cache=npm-cache', 'CI=true', 'NODE_OPTIONS=--max_old_space_size=8192']) {
-    withDockerContainer(image: 'node:16', args: '-u=root -e HOME=/tmp -e NPM_CONFIG_PREFIX=/tmp/.npm') {
+    withDockerContainer(image: 'node:18', args: '-u=root -e HOME=/tmp -e NPM_CONFIG_PREFIX=/tmp/.npm') {
```

Comment:
Needs to be validated in the scheduled release after this PR is merged

Reply:
Eventually, I'm thinking we should switch to use [Volta](https://volta.sh/) instead. However, I recently tried (in my free time) to use Volta in Docker and it was a pain. It worked very well in GitHub Actions and on Windows though.